init_reverse_lookup_registry_v1/sources/script.move

```diff
@@ -0,0 +1,7 @@
+script {
```

Comment:
This can be removed after updating init_reverse_lookup_registry_v1 to be public entry fun

Reply:
I'll update it in the next version.